notebooks/3_informed_search/index.ipynb

```diff
@@ -4,514 +4,281 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Informed Search\n",
+    "## Introduction\n",
     "\n",
-    "Informed search is an approach to solving problems with a start state to a goal state, that is based on some mathematical concepts that estimate our distance to a goal state and inform us about some of following states. \n",
```

Comment:
and rewriting an already written notebook made this reviewing process really hard for me, so thx :)))

Reply:
can you explain this comment? 
did I screw something up with GitHub?
apologies in advance.